var/spack/repos/builtin/packages/highfive/package.py

```diff
@@ -42,12 +42,13 @@ class Highfive(CMakePackage):
 
     def cmake_args(self):
         return [
-            '-DUSE_EIGEN:Bool=' + ('TRUE' if '+eigen' in self.spec else 'FALSE'),
-            '-DUSE_XTENSOR:Bool=' + ('TRUE' if '+xtensor' in self.spec else 'FALSE'),
-            '-DUSE_BOOST:Bool={0}'.format('+boost' in self.spec),
-            '-DHIGHFIVE_PARALLEL_HDF5:Bool={0}'.format('+mpi' in self.spec),
-            '-DHIGHFIVE_EXAMPLES:Bool={0}'.format(self.spec.satisfies('@develop')),
-            '-DHIGHFIVE_UNIT_TESTS:Bool={0}'.format(self.spec.satisfies('@develop')),
-            '-DHIGHFIVE_TEST_SINGLE_INCLUDES:Bool={0}'.format(self.spec.satisfies('@develop')),
+            '-DUSE_BOOST:Bool=' + str(self.spec.satisfies('+boost')),
```

Comment:
just nitpick: its true that `self.spec.satisfies` return true/false but older syntax ('TRUE' if '+eigen' in self.spec else 'FALSE') is common in spack packages.

Reply:
I can make a revision tonight to address all the comments since there are several small things

Reply:
Just checking, it seems both ways are used interchangeably... 
```
leite@bluebrain291 packages $ grep 'in self.spec' * -R | wc -l
     537
leite@bluebrain291 packages $  grep 'spec.satisfies' * -R | wc -l
     472
leite@bluebrain291 packages $ grep "'TRUE' if" * -R | wc -l
       9
```
Now between
`'-DUSE_EIGEN:Bool=' + str(self.spec.satisfies('+eigen'))`
and
`'-DUSE_EIGEN:Bool=' + ('TRUE' if '+eigen' in self.spec else 'FALSE')`
Do you have a strong preference? IMHO +1 for the shortest and the `satisfies` API :)

Reply:
It’s ok.
For the second, I have always seen later.